haiku-apps/midikeyboard/midikeyboard-1.0.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="An onscreen midi keyboard"
+DESCRIPTION="An awesome keyboard on the screen for playing music"
+HOMEPAGE="https://github.com/HaikuArchives/MidiKeyboard"
+COPYRIGHT="1991, 1989 Free Software Foundation"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+srcGitRev="25fad6b3f1207e5c7b62a8aa49b0a775fae71899"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="7fa674575ae0b0b963d152ea36f9802d8d1949bd4bfd9513c6fe1dc33712a891"
+SOURCE_DIR="MidiKeyboard-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	midikeyboard$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
```

Comment:
It looks like the compilation only succeeds on x86_gcc2.
But INSTALL is not doing anything. So I found the runtime in:
```
work-1.0.0/sources/MidiKeyboard-*/objects.x86-gcc2-release/midikeyboard
```
and copied it to /system/non-packaged/bin/

We need to find out how to fix the INSTALL and maybe try to make the app build on x86 and x86_64.